SwiftWisdom/Core/StandardLibrary/Array/Array+Utilities.swift

```diff
@@ -54,3 +54,43 @@ extension Array {
         }
     }
 }
+
+extension CollectionType {
+    public var ip_middleValues: [Generator.Element]? {
```

Comment:
I think in its current implementation, the use of this function is kind of unclear.  It appears to be getting the middle values of the collection, but kind of sounds like it might use comparable to get middle.  I'm also not sure about the array return.  Let's pull this one out while we think about it.